Functional UI using new schema types

ThatTobMate opened the pull request:
This PR moves proplot v2 frontend code into its own directory within the webapp, it can be enabled by setting the `REACT_APP_PROP_LOT_BETA_ENABLED` property on the netlify builds.

The V2 code is powered by the GraphQL schema and we have implemented some tooling to fetch and build the schema to ensure we have type safety when building the UI.

In this PR:

- Define PropLot schema querys for the list and filters
- Migrate PropLot UI to use the GraphQL query
- Move PropLot v2 code into a subdirectory in the web app
- Setup enabling of PropLot V2 with a env var
- Update Filter UI to match the designs

mcgingras commented on packages/nouns-webapp/src/components/PropLotList/PropLotList.module.css:
```diff
@@ -0,0 +1,205 @@
+.heading {
+  font-family: 'Londrina Solid';
+  font-size: 40px;
+  margin: 0 !important;
+}
```
no love for tailwind? :)

ThatTobMate replied:
Ha yes, I've become a fan! This is here as it was just a copy paste of the Ideas component so I could get a rough example up

ThatTobMate commented:
Cleaned up the schema for this spike and added the rendering + application of different filter types. Only Sort filters actually do anything to the list as they're the only ones being parsed on the API in this PR.

All filters are built using the same generic types and handled in the same way, the API can choose how it handles the values applied to the query.

Also moved the proplot specific UI code into it's own directory and added commands to build type definitions for all schema properties.

https://user-images.githubusercontent.com/7782211/191020741-a0954bb5-721f-48d5-a40d-87b88372eca3.mov
